### DISCUSSION TITLE
Grove - IMU 9DOF v2.0 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#972] [Q&A]
# Grove - IMU 9DOF v2.0 | Seeed Studio Wiki

Grove - IMU 9DOF v2.0

https://wiki.seeedstudio.com/Grove-IMU_9DOF_v2.0/

**Comment by tomas-polach** [comment]
On a Raspberry PI, this one repo really got me there with achieving my goals in a practical application: https://github.com/niru-5/imusensor

It helps with calibration, finding roll, pitch and yaw and sensor fusion with Kalman or Madgwick filters. Means: you get the actual stable degrees of orientation (instead of some jittery numbers).
